netbox/dcim/search.py

```diff
@@ -105,7 +105,7 @@ class MACAddressIndex(SearchIndex):
         ('mac_address', 100),
         ('description', 500),
     )
-    display_attrs = ('mac_address', 'interface')
+    display_attrs = ('mac_address', 'assigned_object')
```

Comment:
We can probably omit the MAC address itself, because it's already the primary representation of the object. Including it here is redundant (see below).

```suggestion
    display_attrs = ('assigned_object',)
```

![screenshot](https://github.com/user-attachments/assets/745a1688-8e92-4458-a380-81a5d488f6c8)

Reply:
Ah, good to know. Added `description` too.